pallets/settlement/src/lib.rs

```diff
@@ -586,6 +589,27 @@ decl_module! {
             Ok(())
         }
 
+        /// Edit a venue's signers.
+        /// * `id` specifies the ID of the venue to edit.
+        /// * `signers` specifies the signers to add/remove.
+        /// * `update_type` specifies the update type add/remove of venue.
+        #[weight = <T as Config>::WeightInfo::update_venue_signers(signers.len() as u32)]
+        pub fn update_venue_signers(origin, id: VenueId, signers: Vec<T::AccountId>, update_type: bool) -> DispatchResult {
+            let did = Identity::<T>::ensure_perms(origin)?;
+
```

Comment:
Need to check that the caller `origin` is allowed to manage the venue using the `Self::venue_for_management` function.  `ensure_perms` just checks that the caller's identity allows them to call this extrinsics, which needs to be done too.

Look at how the `add_instruction` extrinsic works.